integration_tests/abstract_account_test.go

```diff
@@ -182,12 +199,18 @@ func TestXionAbstractAccount(t *testing.T) {
 	)
 	require.NoError(t, err)
 
-	updatedContractstate, err := ExecQuery(t, ctx, xion.FullNodes[0], "wasm", "contract-state", "smart", aaContractAddr, fmt.Sprintf(`{"pubkey":{}}`))
+	updatedContractstate, err := ExecQuery(t, ctx, xion.FullNodes[0], "wasm", "contract-state", "smart", aaContractAddr, fmt.Sprintf(`{"authenticator_by_i_d":{ "id": 0 }}`))
```

Comment:
does this work with the key misspelling?

Reply:
The contract is expecting the key like this `authenticator_by_i_d` becuase the type in the contract is `AuthenticatorByID`.